external/docs/data/docs.yml

```diff
@@ -4,1007 +4,1007 @@ versions:
     commit_sha: 6d7babea1d02a11eb4b4d6dd66e35b569d201fbe
     tree_sha: bc6ce29d1ec757d9d036532531a1046db4da0d96
     committed: 2014-12-17 19:30:46.000000000 +00:00
-    date: 12/17/14
+    date: '2014-12-17'
```

Comment:
I hope that you did not do this by hand but instead let the script do its job...

Reply:
Darn it, I should have realized that this is affected by the v2.49.0 update.

Reply:
I am letting [the workflow](https://github.com/dscho/git-scm.com/actions/runs/13864269831) redo it.

Reply:
I force-pushed the result.

Reply:
I merged it into my own fork's `gh-pages` and deployed it, you can see it in action e.g. [here](https://dscho.github.io/git-scm.com/docs/git-backfill)

![image](https://github.com/user-attachments/assets/03b95863-74d7-44d9-9a12-b3afdd587d82)